src/Phantoman.php

```diff
@@ -34,6 +34,11 @@ public function __construct($config, $options)
             $this->config['path'] = "vendor/bin/phantomjs";
         }
 
+        // Add .exe extension if running on the windows
+        if ($this->isWindows() && file_exists(realpath($this->config['path'] . '.exe'))) {
+            $this->config['path'] .= '.exe';
+        }
```

Comment:
Obviously the order was wrong. Now it works as you described. I've tested with `vendor/bin`, `vendor/bin/phantomjs`, `vendor/bin/phantomjs.exe` and empty (using the default value). All paths **works** as expected.

So I think you can merge this.